modules/end-user-guide/nav.adoc

```diff
@@ -1,49 +1,49 @@
 .End-user Guide
 
-* xref:navigating-che.adoc[]
-** xref:navigating-che-using-the-dashboard.adoc[]
-** xref:importing-certificates-to-browsers.adoc[]
-** xref:accessing-che-from-openshift-developer-perspective.adoc[]
-* xref:che-theia-ide-basics.adoc[]
-** xref:defining-custom-commands-for-che-theia.adoc[]
-** xref:version-control.adoc[]
-** xref:che-theia-troubleshooting.adoc[]
-** xref:differences-in-how-che-theia-webview-works-on-a-single-host-mode-comparing-to-a-multi-host-mode.adoc[]
+// * xref:navigating-che.adoc[]
+// ** xref:navigating-che-using-the-dashboard.adoc[] // NOT NECESSARY
+// ** xref:importing-certificates-to-browsers.adoc[] // OBSOLETE
+* xref:accessing-che-from-openshift-developer-perspective.adoc[]
+// * xref:che-theia-ide-basics.adoc[] // REDIRECT TO https://theia-ide.org/docs/
+// ** xref:defining-custom-commands-for-che-theia.adoc[] // REDIRECT TO https://devfile.io/docs/devfile/2.0.0/user-guide/adding-commands-to-a-devfile/
```

Comment:
should this be to latest standard, https://devfile.io/docs/devfile/2.1.0/user-guide/adding-commands-to-a-devfile/ ?

Reply:
Yes